# Add linux/err.h for MAX_ERRNO definition, needed by util-linux mount

Index: linux-2.6.19-rc1/usr/include/linux/err.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ linux-2.6.19-rc1/usr/include/linux/err.h	2006-10-05 10:55:17.000000000 +0200
@@ -0,0 +1,16 @@
+#ifndef _LINUX_ERR_H
+#define _LINUX_ERR_H
+
+#include <asm/errno.h>
+
+/*
+ * Kernel pointers have redundant information, so we can use a
+ * scheme where we can return either an error code or a dentry
+ * pointer with the same return value.
+ *
+ * This should be a per-architecture thing, to allow different
+ * error and pointer decisions.
+ */
+#define MAX_ERRNO	4095
+
+#endif /* _LINUX_ERR_H */
